pwndbg/dbg/gdb.py

```diff
@@ -452,10 +453,37 @@ def selected_frame(self) -> pwndbg.dbg_mod.Frame | None:
             pass
         return None
 
+    def commands(self):
+        current_pagination = gdb.execute("show pagination", to_string=True)
```

Comment:
Fwiw
```
(.venv) root@pwndbg:~/pwndbg# rg 'set pagination'
pwndbg/dbg/gdb.py
232:        set pagination off
```

But sure, user can change it

Reply:
I just copied the implementation from `pwndbg/commands/__init__.py`, felt like I didn't have much of a reason to change it